backend/README.md

```diff
@@ -0,0 +1,37 @@
+# Build Backend:
```

Comment:
Can you add a section describing how to run the backend? ie what commands  to run.